arkouda/pdarrayclass.py

```diff
@@ -2236,6 +2238,56 @@ def sum(pda: pdarray) -> np.float64:
     return parse_single_value(cast(str, repMsg))
 
 
+@typechecked
+def dot(
+    pda1: Union[np.int64, np.float64, np.uint64, pdarray],
+    pda2: Union[np.int64, np.float64, np.uint64, pdarray],
+) -> Union[np.int64, np.float64, np.uint64, pdarray]:
+    """
+    Returns the sum of the product of two arrays of the same size (the dot product) or
```

Comment:
maybe we reword this to be:
```suggestion
    Returns the sum of the elementwise product of two arrays of the same size (the dot product) or
```
because otherwise the product of 2 arrays is a bit ambiguous

Reply:
I agree, that is much more clear. Fixed